### PR TITLE
Vc landing page

### PR DESCRIPTION
**Pull request by bhanuprakash-egov**
@saiprakash-v @prasanna-egov

### CODE REVIEW

File: tenant-portal/src/components/Welcome/Welcome.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        navigate('/tenant-portal/vcwelcome');
+    };
+    const manageSchema = () => {
+        navigate('/tenant-portal/vcwelcome');
+    };
+    return(
+        <div className="row m-4">
+            <div className="col-md-6">
+                <div className="p-2">
+                    <h2>Welcome to DIVOC VC Issuance platform</h2><br/>
```
Comment (prasanna-egov) [inline tenant-portal/src/components/Welcome/Welcome.js:20]: All this text should be internationalized. It should come from language file.

File: tenant-portal/src/components/Header/Header.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+function Header() {
+  return (
+    <div>Header</div>
```
Comment (prasanna-egov) [inline tenant-portal/src/components/Header/Header.js:5]: Where is the content for the header.

**Review by prasanna-egov — APPROVED** [review]

File: tenant-portal/src/components/Welcome/Welcome.js (hunk 1 above)
Comment (bhanuprakash-egov) [reply tenant-portal/src/components/Welcome/Welcome.js:20]: okay will change it

File: tenant-portal/src/components/Header/Header.js (hunk 2 above)
Comment (bhanuprakash-egov) [reply tenant-portal/src/components/Header/Header.js:5]: content is from vishwa HeaderComponent pr, will update it

**Pull request closed without merging**